VultisigApp/VultisigApp/Services/Protobuf/ProtoSerializer.swift

```diff
@@ -14,7 +14,7 @@ struct ProtoSerializer {
     static func serialize<T: ProtoMappable>(_ model: T) throws -> String {
         let proto = model.mapToProtobuff()
         let compressed = try proto.serializedData()
-        let compressedData = try (compressed as NSData).compressed(using: .zlib)
+        let compressedData = try (compressed as NSData).compressed(using: .lzma)
         let compressedDataBase64 = compressedData.base64EncodedString()
```

Comment:
Change this will break interop with android devices

Reply:
I'll ping android team